fe/fe-core/src/test/java/com/starrocks/analysis/LoadStmtTest.java

```diff
@@ -88,4 +88,10 @@ public void testBrokerLoad() {
         analyzeSuccess("LOAD LABEL test.testLabel (DATA INFILE(\"hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file\") INTO TABLE `t0`) WITH BROKER (\"username\"=\"sr\") PROPERTIES (\"strict_mode\"=\"true\")");
         analyzeSuccess("LOAD LABEL test.testLabel (DATA INFILE(\"hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file\") INTO TABLE `t0`) WITH BROKER (\"username\"=\"sr\")");
     }
+
+    @Test
+    public void testToString() {
+        LoadStmt stmt = (LoadStmt) analyzeSuccess("LOAD        LABEL test.testLabel (DATA INFILE(\"hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file\") INTO TABLE `t0`) WITH BROKER hdfs_broker (\"username\"=\"sr\", \"password\"=\"PASSWORDDDD\") PROPERTIES (\"strict_mode\"=\"true\")");
+        Assert.assertEquals("LOAD LABEL `test`.`testLabel`(DATA INFILE ('hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file') INTO TABLE t0)WITH BROKER hdfs_broker (\"password\"  =  \"*XXX\", \"username\"  =  \"sr\")PROPERTIES (\"strict_mode\" = \"true\")", stmt.toString());
```

Comment:
What's the `*XXX`?  You can use `xxx` instead.

Reply:
https://github.com/StarRocks/starrocks/blob/a1c41554cceced937e3cd4fb3a57d3a9485c567c/fe/fe-core/src/main/java/com/starrocks/common/util/PrintableMap.java#L87-L89
It's added by the `PrintableMap.toString`. How about replacing it with `***` ?